src/Controls/src/Core/Layout.cs

```diff
@@ -468,6 +469,7 @@ void InternalChildrenOnCollectionChanged(object sender, NotifyCollectionChangedE
 
 		void OnInternalAdded(View view)
 		{
+			TypeHashCode = null;
```

Comment:
These 2 should not be needed as any change to `InternalChildren` will fire InternalChildrenOnCollectionChanged.